sklearn/ensemble/tests/test_bagging.py

```diff
@@ -288,7 +288,8 @@ def test_probability():
                                   np.exp(ensemble.predict_log_proba(X_test)))
 
         # Degenerate case, where some classes are missing
-        ensemble = BaggingClassifier(base_estimator=LogisticRegression(),
+        ensemble = BaggingClassifier(base_estimator=LogisticRegression(
+                                          solver='lbfgs'),
```

Comment:
Unsure if this is the best way to get around the failure here: https://github.com/scikit-learn/scikit-learn/pull/5204#issuecomment-137265512, but it doesn't appear to break anything.